elasticizer/pipelines/med_search.py

```diff
@@ -0,0 +1,186 @@
+import os
+import json
+import random
+import luigi
+from luigi.postgres import PostgresQuery
+from luigi.contrib.esindex import ElasticsearchTarget, CopyToIndex
+import elasticizer
+
+fields = [
+  'id_cfpb',
+  'name',
+  'name_short',
+  'active_date',
+  'inactive_date',
+  'inst_type_code',
+  'inst_status_code',
+  'fax_number',
+  'phone_number'
+  ]
+
+# -----------------------------------------------------------------------------
+# Tasks
+# -----------------------------------------------------------------------------
+
+class Extract(PostgresQuery):
+    host = os.getenv('DB_HOST', '127.0.0.1')
+    database = os.getenv('DB_NAME', 'mydb')
+    user = os.getenv('PGUSER', 'vagrant')
+    password = os.getenv('PGPASSWORD', 'vagrant')
+    table = 'institution'
+    query = 'SELECT * FROM institution'
+
+
+class Format(luigi.Task):
+    index = luigi.Parameter()
+
+    def _projection(self, row):
```

Comment:
Consider `_label_row` instead of `_projection` ? Also, I'd pull out the `date` bit since that's a lot of mixed up functionality in one line.